ansible/config_sonic_basedon_testbed.yml

```diff
@@ -83,10 +83,7 @@
     when: ("'host_interfaces' in vm_topo_config") and ("'tor' in vm_topo_config['dut_type'] | lower")
 
   - name: find all vlan configurations for T0 topology
-    vlan_config: 
-      vm_topo_config: "{{ vm_topo_config }}"
-      port_alias: "{{ port_alias }}"
-      vlan_config: "{{ vlan_config | default(None) }}"
```

Comment:
Were you not able to simply remove the quotes?

I prefer the proper YAML syntax, and it was I who provided you with an example using quotes. I took that from an example where the variables were strings. I'm certain there's a way to use the YAML syntax and still retain the variable as a dictionary.

Reply:
After researching, it appears it is not possible:

https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#hey-wait-a-yaml-gotcha

Reply:
It's extra-strange that this syntax works in older versions of Ansible (2.0.0.2) but not in 2.8.7.